docs/docs/7-migration-guide/migration-from-v6-to-v7.md

```diff
@@ -1080,6 +1061,25 @@ const nextConfig = {
 module.exports = withBundleAnalyzer(nextConfig);
 ```
 
+### Adjust CDN config in `site/server.js`
+
+```diff
+// site/server.js
+
+- const cdnEnabled = process.env.CDN_ENABLED === "true";
+- const disableCdnOriginHeaderCheck = process.env.DISABLE_CDN_ORIGIN_HEADER_CHECK === "true";
+- const cdnOriginHeader = process.env.CDN_ORIGIN_HEADER;
++ const cdnOriginCheckSecret = process.env.CDN_ORIGIN_CHECK_SECRET;
+
+// ...
+
+- if (cdnEnabled && !disableCdnOriginHeaderCheck) {
+-    const incomingCdnOriginHeader = req.headers["x-cdn-origin-check"];
+-    if (cdnOriginHeader !== incomingCdnOriginHeader) {
++ if (cdnOriginCheckSecret) {
++    if (req.headers["x-cdn-origin-check"] !== cdnOriginCheckSecret) {
+```
+
```

Comment:
We should directly migrate to the version from the starter where we removed `DISABLE_CDN_ORIGIN_HEADER_CHECK`. See https://github.com/vivid-planet/comet-starter/blob/main/site/server.ts#L25

Reply:
I believe it already is: https://deploy-preview-3502.docs.comet-dxp.com/docs/migration-guide/migration-from-v6-to-v7#adjust-cdn-config-in-siteserverjs

Reply:
yes, my bad. Kind of hard 😅